config/jobs/kubernetes/kops/templates/presubmit-scenario.yaml.jinja

```diff
@@ -20,6 +20,11 @@
     decoration_config:
       timeout: {{job_timeout}}
     path_alias: k8s.io/kops
+    extra_refs:
+    - org: kubernetes
+      repo: perf-tests
+      base_ref: master
+      path_alias: k8s.io/perf-tests
```

Comment:
This is specific to perf-tests, but the template is for all kOps pre-submits. Maybe check if scenario is "scalability"?

Reply:
Thanks good call, I have added this check.